test/acceptance/features/bindAppToServiceWithLabelSelector.feature

```diff
@@ -0,0 +1,111 @@
+Feature: Bind an application with label selector to a service
+
+    As a user of Service Binding Operator
+    I want to bind applications with label selector to services it depends on
+
+    Background:
+        Given Namespace [TEST_NAMESPACE] is used
+        * Service Binding Operator is running
+
+    Scenario: Bind an application with label selector to backend service
+        Given Generic test application "gen-app-a-s-c" is running
+        * CustomResourceDefinition backends.stable.example.com is available
+        * The Secret is present
+            """
+            apiVersion: v1
+            kind: Secret
+            metadata:
+                name: backend-secret
+            stringData:
+                username: AzureDiamond
+                password: hunter2
+            """
+        And The Custom Resource is present
+            """
+            apiVersion: stable.example.com/v1
+            kind: Backend
+            metadata:
+                name: service-a-s-c
+                annotations:
+                    service.binding: path={.status.data.dbCredentials},objectType=Secret,elementType=map
+            status:
+                data:
+                    dbCredentials: backend-secret
+            """
+        When Service Binding is applied
+            """
+            apiVersion: binding.operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: service-binding-a-s-c
+            spec:
+                bindAsFiles: false
+                services:
+                  - group: stable.example.com
+                    version: v1
+                    kind: Backend
+                    name: service-a-s-c
+                application:
+                    labelSelector:
+                      matchLabels:
+                        app: gen-app-a-s-c
+                    group: apps
+                    version: v1
+                    resource: deployments
+            """
+        Then Service Binding "service-binding-a-s-c" is ready
+        And The application env var "BACKEND_USERNAME" has value "AzureDiamond"
+        And The application env var "BACKEND_PASSWORD" has value "hunter2"
+
+    Scenario: Attempt to bind an application with label selector and name to a service
```

Comment:
please rephrase to include the outcome as well.. in this case service binding will be unsuccessful, right?

Reply:
Done. af6e990baa021c05e9fba2e7cb713ac745500ba9